utils/config.sh

```diff
@@ -0,0 +1,5 @@
+# File with demo configuration parameters
+
+export CP_VERSION_MAJOR=5.4
+export CP_VERSION_FULL=5.4.0
```

Comment:
Would it make sense to derive the major version from the full version to reduce the various required to be managed?

Something like:
`CP_VERSION_MAJOR=${CP_VERSION_FULL%.*}`

A little harder w/ the branch but above maybe saves a substitution

Reply:
@rspurgeon A concern with doing this is I don't remember how previews are managed.  For example, the 5.4 preview (pre-GA).  The MAJOR would be 5.4 (and that's what Confluent CLI would show) but I don't remember what the full would show (as used in other commands).  So while I acknowledge that this optimization reduces the number of lines to manage, it opens potential problems later.  tl;dr the value in keeping them separate is future-proofing against mis-matched naming conventions that we don't know.